.gitpod/readme.md

```diff
@@ -39,29 +39,25 @@ where:
 * Scaffold modules, messages, types with CRUD operations, IBC packets, and more
 * Start a blockchain node in development with live reloading
 * Connect to other blockchains with a built-in IBC relayer
-* Use automatically generated TypeScript/Vuex clients to interact with your blockchain
-* Use the Vue.js web app template with a set of components and Vuex modules
+* Use automatically generated TypeScript clients to interact with your blockchain
```

Comment:
what kind of clients do we auto generate?

Reply:
Let's remove all mentions of Vuex. We can say we're generating a TypeScript client.